### DISCUSSION TITLE
Conversion of .d to .dia

### DISCUSSION THREAD

**Discussion by kbarylyuk** [vdemichev/DiaNN#1134] [Q&A] [answered]
Hi Vadim!

I have a question regarding the conversion of Bruker diaPASEF `.d` raw files to DIA-NN's `.dia`. In the [GUI settings reference](https://github.com/vdemichev/DiaNN?tab=readme-ov-file#gui-settings-reference), it says:

> Conversion is recommended for Sciex files, typically makes little difference for Thermo files and **is not recommended for Bruker files**

Could you please elaborate on why the conversion is not recommended? Is there any detrimental effect to the data? Are the files converted by an older version of DIA-NN forward-compatible with the newer versions?

The reason I am asking is because we have a set of data from timsTOF generated by an external partner that is only available in `.dia` format.

Thank you!

**Comment by vdemichev** [comment, marked as answer]
Hi Konstantin!

That’s fine if it’s dia.

It’s not recommended for two reasons:
- File size might be up to 3x larger for dia, this is due to Bruker using a very effective compression algorithm.
- .dia conversion is not supported for Slice, Synchro and midia-PASEF.

Best,
Vadim

**Reply by kbarylyuk** [reply]
Thank you very much for the prompt reply, Vadim!
Best,
Konstantin
P.S. Long time no see ))
